src/rpc/mempool.cpp

```diff
@@ -247,34 +247,81 @@ static RPCHelpMan testmempoolaccept()
     };
 }
 
+static std::vector<RPCResult> ClusterDescription()
+{
+    return {
+        RPCResult{RPCResult::Type::NUM, "vsize", "virtual transaction size as defined in BIP 141. This is different from actual serialized size for witness transactions as witness data is discounted."},
+        RPCResult{RPCResult::Type::NUM, "txcount", "number of transactions (including this one)"},
+        RPCResult{RPCResult::Type::NUM, "clusterid", "id of the cluster containing this tx"},
```

Comment:
About the `clusterid` (`m_next_cluster_id`), I don’t think it’s a good idea to use it as a tie-breaker in `CompareMiningScore` as it’s just a monotonic counter, it’s not information perfomative for a mining score. You could use weight at least or reception timestamp e.g oldest the tx-relay reception more well-propagated on the network, already in every miner mempool.